src/ibtracert.c

```diff
@@ -811,9 +820,29 @@ int main(int argc, char **argv)
 	argc -= optind;
 	argv += optind;
 
-	if (argc < 2)
+	if (argc < 2 && lid_file[0] == '\0') 
```

Comment:
Should this be:
if (argc < 2 && lid_file == NULL)
?

Reply:
probably